### DISCUSSION TITLE
Autoclose Extension Sites after visit.

### DISCUSSION THREAD

**Discussion by ecxod** [microsoft/vscode-discussions#496] [Extension Show and Tell]
Is there a way to auto close extension sites after visit ?
If I visit 100 extensions they all stay pined on top and I must manually close 100 sites. This is bad UX. Can someone tell if there is a setting to automatically close them except when I pin them up.

**Comment by alefragnani** [comment]
Hi @ecxod ,

This must be happening because you have set `"workbench.editor.enablePreview": false` on your User Settings. If you return to the default value (`true`), it won't be opened, unless you _double click_ it. You will notice the different behavior because the tab title will be displayed _italic_

Be aware this setting controls _every tab/editor_ you open, not only Extensions view.

Hope this helps

**Comment by ecxod** [comment]
Thank you very much, but it is not 100% what i want. I want just that `Extension: ....` Tabs should not stay open. When I click on another Extension the first should close.

**Reply by alefragnani** [reply]
I see. In this case, you should open a feature request in VS Code repo https://github.com/microsoft/vscode/issues.

Hope this helps

**Comment by ecxod** [comment]
I see your project manager for some time, but I never figured out how to use it.
I take a deeper look now :)) 
Have maximum fun

**Reply by alefragnani** [reply]
Thank you! It is one of my first extensions, and coincidentally the most used. 

I'm planning adding an `Walkthrough`, and I hope it could help new users to take full advantage of its features. Feel free to add comments/suggestions in extension's repo (https://github.com/alefragnani/vscode-project-manager/issues), for areas that you miss any info
